README.md

```diff
@@ -92,9 +93,9 @@ It is not needed for recruitment assignments, unless specified otherwise.
 It is needed for trial period assignments.
 
 
-## Sign in to your new e-mail account
+## Sign in to your new email account
```

Comment:
While `e-mail` is technically more correct, across the document `email` is used 9x, in comparison `e-mail` is present only 3x, so the suggested change goes this way.